js/api/modules/member.js

```diff
@@ -0,0 +1,29 @@
+import { api } from "/js/api/index.js";
+import { Member } from "/js/core/member.js";
+import { $loadingBar } from "/js/view/loadingbar.js";
+
+export const memberApi = {
+  async findAllMembers() {
+    $loadingBar.loading();
+    const members = await api.get("/api/users");
+    return members.map((member) => new Member(member));
+  },
+
+  async saveMember(member) {
+    $loadingBar.loading();
+    const savedMember = await api.post("/api/users/", {
+      name: member.name,
+    });
+    return new Member(savedMember);
+  },
```

Comment:
```suggestion
 async saveMember({ name }) {
    $loadingBar.loading();
    const savedMember = await api.post("/api/users/", { name });
    return new Member(savedMember);
  },
```

위와 같이 간단하게 작성하는 것도 가능합니다 ㅎㅎ